src/main/java/bio/terra/cli/context/utils/Printer.java

```diff
@@ -48,14 +43,30 @@ private Printer(PrintStream out, PrintStream err) {
    */
   public static void setupPrinting(CommandLine cmd) {
     if (printer == null) {
-      printer = new Printer();
+      initialize(DEFAULT_OUT_STREAM, DEFAULT_ERR_STREAM);
     } else {
-      logger.warn("Printing setup called multiple times.");
+      logger.warn(
```

Comment:
I generally don't like to use testing flags or modes, because that takes me farther away from testing the "normal operation". Sometimes I don't know a way around it (e.g. the environment variable to override where to put the global context directory), but in this case it's just a log statement.